app/models/manageiq/providers/hawkular/middleware_manager/live_metrics_capture.rb

```diff
@@ -50,8 +50,8 @@ def collect_live_metrics(metrics, start_time, end_time, interval)
         end
         metrics_ids_map[metric_id] = metric[:name]
       end
-      starts = (start_time - interval).to_i.in_milliseconds
-      ends = end_time.to_i.in_milliseconds + 1
+      starts = start_time.to_i.in_milliseconds
+      ends = (end_time + interval).to_i.in_milliseconds + 1
```

Comment:
Previous logic worked that from start, end, it started start -1h to end.
So, in start 00:00, it fetched 23:00 until 23.00.
This fixes the effective interval starting in the real hour, not 1 hour before.